pkg/frontend/transport/handler.go

```diff
@@ -143,8 +143,16 @@ func (f *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	w.WriteHeader(resp.StatusCode)
-	// we don't check for copy error as there is no much we can do at this point
-	_, _ = io.Copy(w, resp.Body)
+	// log copy error so that we will know even though success response code returned, copy response error occurred
+	bytesCopied, err := io.Copy(w, resp.Body)
+	if err != nil {
+		msg := []interface{}{
```

Comment:
I think code looks simpler if we do `...Log("msg", "write response body error", "bytesCopied", bytesCopied, "err", err)` instead of creating a slice of one element.

Reply:
Sounds good to me, updated